src/gretel_synthetics/train.py

```diff
@@ -121,8 +154,13 @@ def create_dataset(store: BaseConfig, text: str, sp: spm.SentencePieceProcessor)
     Create two lookup tables: one mapping characters to numbers,
     and another for numbers to characters.
     """
-    # Create training dataset
-    char_dataset = tf.data.Dataset.from_tensor_slices(sp.EncodeAsIds(text))
+    logging.info("Tokenizing training data")
+    ids = []
+    for line in tqdm(text.split("\n")):
+        ids = ids + sp.EncodeAsIds(line)
```

Comment:
can we `ids.append(sp.EncodeAsIds(line)` here?

Reply:
Need to keep as one contiguous array, but can simplify to `ids += sp.EncodeAsIds(line)`

Reply:
Ah right. Then use extend()